pkg/serviceprovider/github/queries.go

```diff
@@ -91,7 +99,27 @@ type PageInfo struct {
 	EndCursor   string `json:"endCursor"`
 }
 
-func _fetchAll(ctx context.Context, client *graphql.Client, req *graphql.Request, self interface{}, pageInfoFromSelf func() PageInfo, processSelf func()) error {
+func _fetchAll(r *AllAccessibleRepos, ctx context.Context, client *graphql.Client, request string, state *TokenState) error {
+
+	lg := log.FromContext(ctx)
+	err := _fetchAllWithPages(ctx, client, graphql.NewRequest(request), r, func() PageInfo {
+		return r.Viewer.Repositories.PageInfo
+	}, func() {
+		for _, node := range r.Viewer.Repositories.Nodes {
+			state.AccessibleRepos[RepositoryUrl(node.Url)] = RepositoryRecord{ViewerPermission: ViewerPermission(node.ViewerPermission)}
+		}
+	})
+	//TODO how to do lg.Debug ?
+	lg.Info("_fetchAll state.AccessibleRepos.len=" + strconv.Itoa(len(state.AccessibleRepos)))
```

Comment:
it will be printed for each call, so this will be printed twice in logs. Wouldn't it be better to log in `FetchAll` function, so it's printed just once?